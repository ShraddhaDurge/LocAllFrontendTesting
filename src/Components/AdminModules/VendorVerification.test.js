import React from 'react';
import renderer from 'react-test-renderer';
import VendorVerification from './VendorVerification';

it('VendorVerification', () => {
    const tree=renderer.create(<VendorVerification />).toJSON();
    expect(tree).toMatchSnapshot();
})