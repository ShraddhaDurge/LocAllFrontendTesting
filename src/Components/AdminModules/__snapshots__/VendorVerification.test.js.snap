// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VendorVerification 1`] = `
<html
  style={
    Object {
      "height": "100%",
    }
  }
>
  <body
    style={
      Object {
        "height": "100vh",
      }
    }
  >
    <center>
      <div
        className="MuiGrid-root"
        style={
          Object {
            "backgroundColor": "#E3F2FD",
            "height": "100vh",
            "margin": "0px",
            "overflow": "auto",
            "postion": "fixed",
          }
        }
      >
        <center>
          <div
            className="MuiPaper-root MuiPaper-elevation20 MuiPaper-rounded"
            style={
              Object {
                "align": "center",
                "margin": "20px",
                "padding": "20px",
                "position": "relative",
                "width": "150vh",
              }
            }
          >
            <div
              align="center"
              className="MuiGrid-root"
              style={
                Object {
                  "padding": "30px 10px",
                }
              }
            >
              <h5
                className="MuiTypography-root MuiTypography-h5 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
              >
                No Vendors Pending to Verify
              </h5>
              <button
                className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-fullWidth"
                disabled={false}
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#2196F3",
                    "display": "flex",
                    "height": "20%",
                    "justifyContent": "center",
                    "margin": "20px auto",
                    "padding": "4px",
                    "width": "30%",
                  }
                }
                tabIndex={0}
                type="submit"
              >
                <span
                  className="MuiButton-label"
                />
              </button>
            </div>
          </div>
        </center>
      </div>
    </center>
  </body>
</html>
`;
